crates/forge_main_neo/src/domain/state.rs

```diff
@@ -17,11 +16,11 @@ pub struct State {
     pub spinner: ThrobberState,
     pub timer: Option<Timer>,
     pub show_spinner: bool,
-    pub spotlight: SpotlightState,
     pub conversation: ConversationState,
     pub chat_stream: Option<CancelId>,
     pub message_scroll_state: ScrollViewState,
     pub menu: MenuState,
+    pub slash_menu_visible: bool,
```

Comment:
we should only have source of truth to decide if we wish to show the menu or not.